[FileInput]: enable tooltip for invalid icon

defymecobra opened the pull request:
## Problem

The validation icon tooltip in FileInput component was not working when hovering over the red invalid icon. The parent wrapper div had `pointer-events-none` CSS class which blocked all pointer interactions, preventing the tooltip from being triggered.

## Solution

Added an inner wrapper with `pointer-events-auto` around the InvalidIcon component to restore pointer events specifically for the icon. This allows the tooltip to work while keeping the surrounding area click-through for the file drop zone functionality.

How it looks now:
<img width="671" height="301" alt="image" src="https://github.com/user-attachments/assets/3e385130-8f2f-49d1-83ed-37d5f05afe93" />

rorychatt commented on frontend/src/widgets/inputs/FileInputWidget.tsx:
```diff
@@ -381,7 +381,9 @@ export const FileInputWidget: React.FC<FileInputWidgetProps> = ({
         <div className="absolute top-2 right-2 z-20 pointer-events-none">
-          <InvalidIcon message={invalid} />
+          <div className="pointer-events-auto">
+            <InvalidIcon message={invalid} />
```
why not just make it so all invalid icons have the pointer-events-auto instead?

so we don't need to do this custom div.

rorychatt reviewed (changes requested):
see my comment. let me know if it is valid or not

defymecobra commented:
## Changes

Moved `pointer-events-auto` logic directly into the `InvalidIcon` component instead of requiring wrapper divs in every usage.

## Benefits

- Removed 5 wrapper divs across FileInputWidget and TextInput variants
- Simplified code and improved maintainability
- InvalidIcon now works automatically regardless of parent's pointer-events settings

defymecobra commented:
@claude review this
